packages/cli/src/util/index.ts

```diff
@@ -290,15 +290,16 @@ export default class Now extends EventEmitter {
 
     if (
       error.errorCode === 'BUILD_FAILED' ||
-      error.errorCode === 'UNEXPECTED_ERROR'
+      error.errorCode === 'UNEXPECTED_ERROR' ||
+      error.errorCode.includes('BUILD_UTILS_SPAWN_')
```

Comment:
What about changing this to ignore the `errorCode` and check for `errorMessage` instead?

```ts
if (error.errorMessage) {
  return new BuildError({
    message: error.errorMessage,
    meta: {},
  });
}
```

Reply:
Is this `error` object coming from the API?

If so I think we need to update it so it can properly indicate if there was a build error or not.

cc @pvdz who is working on improved error codes

Reply:
It is. I'm happy to revise this for better handling of API errors.